pkgs/games/fallout-ce/fallout2-ce.nix

```
@@ -4,18 +4,20 @@
 }:
 
 callPackage ./build.nix rec {
-  name = "fallout2-ce";
+  pname = "fallout2-ce";
+  version = "1.2.0";
 
   src = fetchFromGitHub {
     owner = "alexbatalov";
-    repo = "fallout2-ce";
-    rev = "v1.2.0";
+    repo = pname;
```

```suggestion
    repo = "fallout2-ce";
```

There's a whole hullaballoo about repo = name https://github.com/nix-community/nixpkgs-lint/issues/21